### PR TITLE
Akka Management 1.0.9

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

File: docs-source/docs/modules/microservices-tutorial/examples/00-shopping-analytics-service-java/src/main/java/shopping/analytics/Main.java [hunk 1]
```diff
@@ -25,8 +25,8 @@ public Main(ActorContext<Void> context) {
     ActorSystem<?> system = context.getSystem();
 
     // FIXME no get(ClassicActorSystemProvider) for Java?
-    AkkaManagement.get(system.classicSystem()).start();
-    ClusterBootstrap.get(system.classicSystem()).start();
+    AkkaManagement.get(system).start();
```
Comment (chbatey) [inline docs-source/docs/modules/microservices-tutorial/examples/00-shopping-analytics-service-java/src/main/java/shopping/analytics/Main.java:28]: remove fix me commend

File: docs-source/docs/modules/microservices-tutorial/examples/00-shopping-order-service-java/src/main/java/shopping/order/Main.java [hunk 2]
```diff
@@ -25,8 +25,8 @@ public Main(ActorContext<Void> context) {
     ActorSystem<?> system = context.getSystem();
 
     // FIXME no get(ClassicActorSystemProvider) for Java?
-    AkkaManagement.get(system.classicSystem()).start();
-    ClusterBootstrap.get(system.classicSystem()).start();
+    AkkaManagement.get(system).start();
```
Comment (chbatey) [inline docs-source/docs/modules/microservices-tutorial/examples/00-shopping-order-service-java/src/main/java/shopping/order/Main.java:28]: ^ here also

File: docs-source/docs/modules/microservices-tutorial/examples/03-shopping-cart-service-java/src/main/java/shopping/cart/Main.java [hunk 3]
```diff
@@ -25,8 +25,8 @@ public Main(ActorContext<Void> context) {
     ActorSystem<?> system = context.getSystem();
 
     // FIXME no get(ClassicActorSystemProvider) for Java?
-    AkkaManagement.get(system.classicSystem()).start();
-    ClusterBootstrap.get(system.classicSystem()).start();
+    AkkaManagement.get(system).start();
```
Comment (chbatey) [inline docs-source/docs/modules/microservices-tutorial/examples/03-shopping-cart-service-java/src/main/java/shopping/cart/Main.java:28]: ^

File: docs-source/docs/modules/microservices-tutorial/examples/shopping-order-service-java/src/main/java/shopping/order/Main.java [hunk 4]
```diff
@@ -25,8 +25,8 @@ public Main(ActorContext<Void> context) {
     ActorSystem<?> system = context.getSystem();
 
     // FIXME no get(ClassicActorSystemProvider) for Java?
-    AkkaManagement.get(system.classicSystem()).start();
-    ClusterBootstrap.get(system.classicSystem()).start();
+    AkkaManagement.get(system).start();
```
Comment (chbatey) [inline docs-source/docs/modules/microservices-tutorial/examples/shopping-order-service-java/src/main/java/shopping/order/Main.java:28]: ^

**Review by chbatey — APPROVED** [review]
LGTM after comment remove

File: docs-source/docs/modules/microservices-tutorial/examples/00-shopping-analytics-service-java/src/main/java/shopping/analytics/Main.java (hunk 1 above)
Comment (patriknw) [reply docs-source/docs/modules/microservices-tutorial/examples/00-shopping-analytics-service-java/src/main/java/shopping/analytics/Main.java:28]: Doh, thx

File: docs-source/docs/modules/microservices-tutorial/examples/00-shopping-analytics-service-java/pom.xml [hunk 5]
```diff
@@ -20,7 +20,7 @@
         <akka.version>2.6.10</akka.version>
         <akka-http.version>10.2.1</akka-http.version>
         <!-- FIXME once akka management 1.0.9 is released -->
-        <akka-management.version>1.0.8+35-9feaa689+20200825-1429</akka-management.version>
```
Comment (ignasi35) [inline docs-source/docs/modules/microservices-tutorial/examples/00-shopping-analytics-service-java/pom.xml:23]: One last FIXME

File: docs-source/docs/modules/microservices-tutorial/examples/shopping-analytics-service-java/pom.xml [hunk 6]
```diff
@@ -20,7 +20,7 @@
         <akka.version>2.6.10</akka.version>
         <akka-http.version>10.2.1</akka-http.version>
         <!-- FIXME once akka management 1.0.9 is released -->
-        <akka-management.version>1.0.8+35-9feaa689+20200825-1429</akka-management.version>
+        <akka-management.version>1.0.9</akka-management.version>
```
Comment (ignasi35) [inline docs-source/docs/modules/microservices-tutorial/examples/shopping-analytics-service-java/pom.xml:23]: another.

**Review by ignasi35 — APPROVED** [review]
two last FIXMEs and we're done. :-)

File: docs-source/docs/modules/microservices-tutorial/examples/00-shopping-analytics-service-java/pom.xml [hunk 7]
```diff
@@ -20,7 +20,7 @@
         <akka.version>2.6.10</akka.version>
         <akka-http.version>10.2.1</akka-http.version>
         <!-- FIXME once akka management 1.0.9 is released -->
```
Comment (ignasi35) [inline docs-source/docs/modules/microservices-tutorial/examples/00-shopping-analytics-service-java/pom.xml:22]:
```suggestion
```

File: docs-source/docs/modules/microservices-tutorial/examples/shopping-analytics-service-java/pom.xml [hunk 8]
```diff
@@ -20,7 +20,7 @@
         <akka.version>2.6.10</akka.version>
         <akka-http.version>10.2.1</akka-http.version>
         <!-- FIXME once akka management 1.0.9 is released -->
```
Comment (ignasi35) [inline docs-source/docs/modules/microservices-tutorial/examples/shopping-analytics-service-java/pom.xml:22]:
```suggestion
```

**Review by ignasi35 — APPROVED** [review]
Raised suggestions to ease the fix.

**Pull request merged**